lib/tasks/trends.rake

```diff
@@ -29,4 +29,57 @@ namespace :trends do
     end
   end
 
+  desc 'Reduces amount of points for each trend group for the last month'
+  task :reduce => :environment do
+    #get the records that sould remain in the DB - should'n be too much of them
+    change_points = ActiveRecord::Base.connection.execute("select trend_id, trend_counters.count, min(created_at) as created_at from trend_counters group by trend_id, count")
```

Comment:
consider using to avoid adapter specific issues
```ruby
TrendCounter.select("trend_id, trend_counters.count, min(created_at) AS created_at").group("trend_id, count")
```